examples/todos/common/build.gradle

```
@@ -0,0 +1,96 @@
+repositories {
+    maven { url "https://dl.bintray.com/spekframework/spek-dev" }
+}
+apply plugin: 'java'
+apply plugin: 'kotlin-multiplatform'
+
+
+kotlin {
+    jvm()
+    js() {
+        [compileKotlinJs, compileTestKotlinJs].each { configuration ->
+            configuration.kotlinOptions {
+                moduleKind = 'umd'
+                sourceMap = true
+                metaInfo = true
+            }
+        }
+    }
+
+
+    iosArm64("ios")
+    iosX64("iosSim")
+    macosX64("macos")
+    mingwX64("win")
+    wasm32("wasm")
+    linuxArm32Hfp("linArm32")
+    linuxMips32("linMips32")
+    linuxMipsel32("linMipsel32")
+    linuxX64("lin64")
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation kotlin("stdlib-common")
+                implementation project(":lib")
+            }
+        }
+        commonTest {
+            kotlin.srcDir('src/test/kotlin')
+            dependencies {
+                implementation kotlin("test-common")
+                implementation kotlin("test-annotations-common")
+                implementation "org.spekframework.spek2:spek-dsl-metadata:$project.versions.spek"
```

with the buildsrc plugin, this can just be
```suggestion
                implementation "org.spekframework.spek2:spek-dsl-metadata:$Versions.spek"
```

---

done